Lib/urllib/request.py

```diff
@@ -2781,3 +2781,30 @@ def proxy_bypass(host):
     # By default use environment variables
     getproxies = getproxies_environment
     proxy_bypass = proxy_bypass_environment
+
+
+if __name__ == "__main__":
+    from argparse import ArgumentParser
```

Comment:
Please turn this into a helper method, e.g. ``def main()``.

Reply:
Done

Reply:
Is it common practice (or impolite) to resolve a fixed issue?